### DISCUSSION TITLE
How to resize partition/ recognize larger partition

### DISCUSSION THREAD

**Discussion by ReDnAx1991** [IceWhaleTech/CasaOS#1790] [Q&A] [answered]
Hi all,

I run CasaOS on a VM with Debian 12 within Proxmox. I have enabled two drives scsi0 (32GB) and scsi1 (1,8TB).
I ran out of storage capacity and increased the scsi1 from 1,8TB to 2,7TB. 

With parted I increased the partition size of /dev/sdb1 to the full 2,7TB, however when I have a look in CasaOS it sees that the drive has been increased to 2,7TB but still shows the 1,8TB as storage capacity.

I tried removing and mounting the storage using the CasaOS GUI without success. Does anyone know how to fix this?

Extra info: 

NAME   MAJ:MIN RM  SIZE RO TYPE  MOUNTPOINTS
sda          8:0     0          32G  0   disk
├─sda1   8:1     0          31G  0   part    /
├─sda2   8:2     0             1K  0   part
└─sda5   8:5     0        975M  0   part   [SWAP]
sdb          8:16   0           2.7T  0  disk
└─sdb1   8:17   0           2.7T  0  part    /mnt/Data_sdb1

Kind regards,
ReD...

**Comment by ReDnAx1991** [comment, marked as answer]
Found the answer via Chat.GPT. I had to sudo resize2fs /dev/sdb1
After everything I did with parted.

In parted I did the following:
- sudo parted /dev/sdb1
- print
- Fix (as parted noticed that there was unused space)
- quit

Restart system
Login
sudo resize2fs /dev/sdb1

**Reply by sund3RRR** [reply]
Thank you for your research!
